engine/render/src/render/render_script.cpp

```diff
@@ -1739,6 +1739,9 @@ namespace dmRender
      * `constants`
      * : [type:constant_buffer] optional constants to use while rendering
      *
+     * `sort_order`
+     * : [type:int] How to sort draw order for world-ordered entries. Default is render.SORT_BACK_TO_FRONT.
+     *
```

Comment:
If I understand idea right, we want to specify sorting order as `render.draw()` function parameter: 
```lua
    draw_options_world.sort_order = render.SORT_FRONT_TO_BACK
    render.draw(predicates.model, draw_options_world)
```